ui/src/helpers/traceback.ts

```diff
@@ -0,0 +1,7 @@
+export const getExceptionLineNumber = (tb: string): number => {
```

Comment:
This is my shitty hack to get line numbers out of the Python traceback. I was looking at Python's built-in `traceback` module but didn't want to spend too much time digging into formatting tracebacks on the Python side before sending over to the JS side. So this felt much lightweight for now!

Reply:
Yeah that seems fine for now. I'm a little concerned with how callbacks will effect the trace stack, but I'm with just seeing what happens